Mathlib/Combinatorics/SimpleGraph/Connectivity/WalkCounting.lean

```diff
@@ -178,7 +178,7 @@ lemma odd_card_odd_components (ho : Odd (Fintype.card V)) :
   by_contra! hc
   simp_rw [← Nat.odd_iff_not_even] at hc
   rw [
-    ← (set_fintype_card_eq_univ_iff _).mpr (@ConnectedComponent.iUnion_supp _ G),
+    ← (set_fintype_card_eq_univ_iff _).mpr (ConnectedComponent.iUnion_supp G),
```

Comment:
```suggestion
    ← (set_fintype_card_eq_univ_iff _).mpr G.ConnectedComponent.iUnion_supp,
```
works hopefully. If it doesn't try renaming the lemma to something out of the `ConnectedComponent` namespace

Reply:
Needed tha latter, done:)